spec/components/works/add_files_component_spec.rb

```diff
@@ -30,7 +30,16 @@
   context 'when globus section' do
     it 'shows the globus upload option' do
       expect(rendered.to_html).to include('Set up a Stanford Globus account')
-      expect(rendered.to_html).not_to include('I have uploaded my files to a Globus endpoint')
+      expect(rendered.to_html).not_to include('Check this box once all your files have been uploaded to Globus.')
```

Comment:
Updating language in spec to match UI (not directly related to changes in this PR).